### PR TITLE
#28 Update UX/UI Markdown (Add Title, Static Content)

### PR DESCRIPTION
**Pull request by trinhlaosu**
- Add package : marked
- Update Editor UI

### CODE REVIEW

**Comment by chiro-hiro** [comment]
This task too big, please broke down. Please, use this format for merge request or commit name

#[Task number] [Title]

File: dmd-frontend/src/components/Editor.tsx [hunk 1]
```diff
@@ -18,7 +21,11 @@ const useStyles = makeStyles((theme: Theme) => ({
     padding: '10px',
     fontFamily: 'Consolas,Courier,serif',
     lineHeight: '21px;',
-    boxSizing: 'border-box',
+    boxSizing: 'border-box'
+  },
+  grid: {
```
Comment (chiro-hiro) [inline dmd-frontend/src/components/Editor.tsx:26]: Please consider it could be an issue, please check to make sure it's mobile responsible.

Comment (trinhlaosu) [reply dmd-frontend/src/components/Editor.tsx:26]: > This task too big, please broke down. Please, use this format for merge request or commit name
> 
> #[Task number] [Title]

Ok, thank you

Comment (trinhlaosu) [reply dmd-frontend/src/components/Editor.tsx:26]: Ok, thank you for suggestions

Comment (chiro-hiro) [reply dmd-frontend/src/components/Editor.tsx:26]: @trinhlaosu I didn't see any changes?

Comment (trinhlaosu) [reply dmd-frontend/src/components/Editor.tsx:26]: I checked and  it's mobile responsible

**Pull request merged**